src/test/scala/day4/conditions.scala

```diff
@@ -0,0 +1,13 @@
+package day4
+import chisel3._
+import org.scalatest._
+import chiseltest._
+class whentest extends FreeSpec with ChiselScalatestTester{

```

Comment:
Please use same class name for test as module name simply place test after that, and capitalize the names.